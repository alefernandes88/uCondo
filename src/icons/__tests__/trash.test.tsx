import React from 'react'
import { render } from '@testing-library/react-native'
import TrashIcon from '../trash'

describe('Icons -> Confirm', () => {
  it('should render with a normal style', () => {
    const { toJSON } = render((
      <TrashIcon />
    ))

    expect(toJSON()).toMatchSnapshot()
  })

  it('should render with a custom style', () => {
    const { toJSON } = render((
      <TrashIcon
        color="red"
        height={30}
        width={30}
        strokeSize={1.5}
      />
    ))

    expect(toJSON()).toMatchSnapshot()
  })
})
