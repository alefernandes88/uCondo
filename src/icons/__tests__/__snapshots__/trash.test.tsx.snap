// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons -> Confirm should render with a custom style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="30"
  bbWidth="30"
  fill="none"
  focusable={false}
  height={30}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 30,
        "width": 30,
      },
    ]
  }
  vbHeight={20}
  vbWidth={20}
  width={30}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M2.5 5H4.16667H17.5"
      propList={
        [
          "stroke",
          "strokeWidth",
          "strokeLinecap",
          "strokeLinejoin",
        ]
      }
      stroke={
        {
          "payload": 4294901760,
          "type": 0,
        }
      }
      strokeLinecap={1}
      strokeLinejoin={1}
      strokeWidth="1.5"
    />
    <RNSVGPath
      d="M6.66668 5.00002V3.33335C6.66668 2.89133 6.84227 2.4674 7.15484 2.15484C7.4674 1.84228 7.89132 1.66669 8.33335 1.66669H11.6667C12.1087 1.66669 12.5326 1.84228 12.8452 2.15484C13.1578 2.4674 13.3333 2.89133 13.3333 3.33335V5.00002M15.8333 5.00002V16.6667C15.8333 17.1087 15.6578 17.5326 15.3452 17.8452C15.0326 18.1578 14.6087 18.3334 14.1667 18.3334H5.83335C5.39132 18.3334 4.9674 18.1578 4.65483 17.8452C4.34227 17.5326 4.16668 17.1087 4.16668 16.6667V5.00002H15.8333Z"
      propList={
        [
          "stroke",
          "strokeWidth",
          "strokeLinecap",
          "strokeLinejoin",
        ]
      }
      stroke={
        {
          "payload": 4294901760,
          "type": 0,
        }
      }
      strokeLinecap={1}
      strokeLinejoin={1}
      strokeWidth="1.5"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`Icons -> Confirm should render with a normal style 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="20"
  bbWidth="20"
  fill="none"
  focusable={false}
  height={20}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 20,
        "width": 20,
      },
    ]
  }
  vbHeight={20}
  vbWidth={20}
  width={20}
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M2.5 5H4.16667H17.5"
      propList={
        [
          "stroke",
          "strokeWidth",
          "strokeLinecap",
          "strokeLinejoin",
        ]
      }
      stroke={
        {
          "payload": 4291085521,
          "type": 0,
        }
      }
      strokeLinecap={1}
      strokeLinejoin={1}
      strokeWidth="1.5"
    />
    <RNSVGPath
      d="M6.66668 5.00002V3.33335C6.66668 2.89133 6.84227 2.4674 7.15484 2.15484C7.4674 1.84228 7.89132 1.66669 8.33335 1.66669H11.6667C12.1087 1.66669 12.5326 1.84228 12.8452 2.15484C13.1578 2.4674 13.3333 2.89133 13.3333 3.33335V5.00002M15.8333 5.00002V16.6667C15.8333 17.1087 15.6578 17.5326 15.3452 17.8452C15.0326 18.1578 14.6087 18.3334 14.1667 18.3334H5.83335C5.39132 18.3334 4.9674 18.1578 4.65483 17.8452C4.34227 17.5326 4.16668 17.1087 4.16668 16.6667V5.00002H15.8333Z"
      propList={
        [
          "stroke",
          "strokeWidth",
          "strokeLinecap",
          "strokeLinejoin",
        ]
      }
      stroke={
        {
          "payload": 4291085521,
          "type": 0,
        }
      }
      strokeLinecap={1}
      strokeLinejoin={1}
      strokeWidth="1.5"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
